src/Views/Content.elm

```diff
@@ -46,22 +47,39 @@ view model =
                         ]
                     , p [] [ a [ class "link link--plain", href "https://neontribe.co.uk/blog" ] [ text "Read in depth about this project" ] ]
                     , div [ class "button-group" ]
-                        [ div [ class "desktop-only" ]
-                            [ div []
-                                [ span []
-                                    [ getIcon "phone" (Just "button--icon")
-                                    , span [] [ text "Call Us" ]
+                        [ if callToAction.action == CallToAction.Phone then
+                            div [ class "desktop-only" ]
+                                [ div []
+                                    [ span []
+                                        [ getIcon callToAction.icon (Just "button--icon")
+                                        , span [] [ text callToAction.promptLong ]
+                                        ]
+                                    , span []
+                                        [ text callToAction.displayHref ]
                                     ]
-                                , span [] [ text helplineDisplay ]
                                 ]
-                            ]
+
+                          else
+                            div [] []
```

Comment:
Had a quick look about this, seems we could use text "" instead to not make a new div,

Reply:
Neat - I guess that also weems a bit odd - but does create preferred markup - of nothing.